ci-operator/step-registry/openshift/microshift/infra/iso-build/openshift-microshift-infra-iso-build-commands.sh

```diff
@@ -54,6 +54,20 @@ if [[ "${JOB_NAME}" =~ .*-cache.* ]] ; then
         ocpversion="4.$(cut -d'.' -f2 "${src_path}/Makefile.version.$(uname -m).var")"
         bash -x ./scripts/fetch_tools.sh brew
         bash -x ./test/bin/manage_brew_rpms.sh download "${ocpversion}" "${out_path}"
+
+        # Fetch brew RPMs for release regression testing
+        # Condition to skip it if manage_brew_rpms.sh script latest version is not backported to all release branches
+        if ( bash ./test/bin/manage_brew_rpms.sh -h | grep '<version_type>' ) ; then
```

Comment:
We usually use `-q` option to hide grep output which is less useful here.
```suggestion
        if ./test/bin/manage_brew_rpms.sh -h | grep -q '<version_type>'  ; then
```